src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,82 @@
+.product-card {
+  width: 200px;
+  background-color: white;
+  border-radius: 5px;
+  border: 1px #f3f3f3 solid;
+  box-sizing: border-box;
+  padding: 33px 16px 16px;
+
+  &__image {
+    width: 160px;
+    display: block;
+    margin: auto;
+  }
+
+  &__name {
+    margin-top: 40px;
+    font-size: inherit;
+    font-weight: 500;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    margin-top: 4px;
+
+    @include secondaryFontSize;
+  }
+
+  &__reviews {
+    margin-top: 16px;
+
+    @include flexSpace;
+
+    &-grades {
+      display: flex;
+      align-items: flex-end;
+
+      @include secondaryFontSize;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+
+    @include flexSpace;
+
+    &-name {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
```

Comment:
it's the default value for `p` tag, so we can easily remove it

Reply:
I deleted the font-size - everything is ok, but if I remove the font-weight, discrepancies with the layout appear. So I left it.